lib/crds/redirects.rb

```diff
@@ -37,12 +37,14 @@ def auth_required
     pages + logins
   end
 
-  def to_csv!(path = './redirects.csv')
+  def to_csv!(path = './redirects.csv', debug=true)
```

Comment:
Was this `debug=true` intentionally left?

Reply:
Yes, I added this flag so I could reduce output in the test suite. Something about the colorized string utility was throwing goofy issues when running the test locally so this seemed like a sensible addition.

Reply:
Makes sense